TestDB.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import sys; sys.path.append('../')
+import faker
+
+class TestUser(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
+        self.faker = faker.Faker()
+        faker = self.faker
+
+    def test_save_customer(self):
+        customer = Customer()
+        customer.first_name = faker.first_name()
+        customer.last_name = faker.last_name()
+        customer.date_created = faker.date()
+        user = User()
+        customer = user.save(customer)
```

Comment:
Where we are now is that conceptually, every entity in our ERD will be a class. For the constraints of our work during this sprint, we will create only the necessary entities qua classes rather than building out the entire ERD. Each entity class will inherit from a parent class that will hold all the database interaction methods (e.g., save, get, etc.).